extensions/ohmyzsh-git-alias/package.json

```diff
@@ -5,6 +5,7 @@
   "description": "Quick search git alias.",
   "icon": "icon.png",
   "author": "yufeikang",
+  "owner": "kang",
```

Comment:
You shouldn't need the `owner` field unless youre an organization

Reply:
I found this. I have removed this. [commit](https://github.com/raycast/extensions/pull/4062/commits/6d3bac8ad9a4104e6040da8d0e5bca0ccfb43cd3)